crates/bevy_text/src/font_atlas_set.rs

```diff
@@ -158,9 +172,54 @@ impl FontAtlasSet {
     pub fn len(&self) -> usize {
         self.font_atlases.len()
     }
-
-    /// Returns `true` if the font atlas set contains no elements
+    /// Returns the number of font atlases in this set
     pub fn is_empty(&self) -> bool {
-        self.font_atlases.is_empty()
+        self.font_atlases.len() == 0
+    }
+
+    /// Get the texture of the glyph as a rendered image, and its offset
+    pub fn get_outlined_glyph_texture(
+        font_system: &mut cosmic_text::FontSystem,
+        swash_cache: &mut cosmic_text::SwashCache,
+        physical_glyph: &cosmic_text::PhysicalGlyph,
+    ) -> (Image, IVec2) {
```

Comment:
IMO we should return a `Result` here. I'd prefer to do this in the current PR, but we can make a follow-up too.